### PR TITLE
Add more info about watchdog notification

### PR DESCRIPTION
**Pull request by dragoangel**

### CODE REVIEW

**Comment by andryyy** [comment]
Thanks. Could you please add this to generate_config, too?

File: generate_config.sh [hunk 1]
```diff
@@ -233,8 +233,11 @@ USE_WATCHDOG=n
 
 ALLOW_ADMIN_EMAIL_LOGIN=n
 
-# Send notifications by mail (no DKIM signature, sent from watchdog@MAILCOW_HOSTNAME)
-# Can by multiple rcpts, NO quotation marks
+# Send notifications by mail (sent from watchdog@MAILCOW_HOSTNAME)
+# CAUTION: 1. USE ONLY EXTERNAL EMAILS AS RECIPIENTS
```
Comment (MAGICCC) [inline generate_config.sh]: I would do a linebreak here imho

**Comment by MAGICCC** [comment]
@andryyy Well he added it to the generate_config script 🤔

**Comment by andryyy** [comment]
Oops. You are correct.

Thought about update.sh, but that's too much hassle to update for this.

File: generate_config.sh (hunk 1 above)
Comment (dragoangel) [reply generate_config.sh]: Done

**Comment by dragoangel** [comment]
@MAGICCC yes I add it to generate new config only. You want that I write some sed in to update.sh to add this notes to existing configs if they not exist?

**Pull request merged**